BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Models/ResourceModel.cs

```diff
@@ -153,5 +154,14 @@ private AttributeModel GetNativeOsVersion()
             }
             return new AttributeModel("os.version", osVersion);
         }
+
+        private AttributeModel GetAndroidSdkInt()
+        {
+            if (Application.platform == RuntimePlatform.Android)
+            {
+                return new AttributeModel("bugsnag.device.android_api_version",AndroidNative.GetAndroidSDKInt());
```

Comment:
formatting nit: missing space?
```suggestion
                return new AttributeModel("bugsnag.device.android_api_version", AndroidNative.GetAndroidSDKInt());
```